Number_types/include/CGAL/Quotient.h

```diff
@@ -44,7 +46,28 @@ namespace CGAL {
 // This function is not documented as a number type requirement for now.
 template < typename NT >
 inline void
-simplify_quotient(NT &, NT &) {}
+simplify_quotient(NT & a, NT & b) { }
+
+#if defined(CGAL_USE_CPP_INT) || true // test cpp_int
+
+inline void
+simplify_quotient(boost::multiprecision::cpp_int & a, boost::multiprecision::cpp_int & b) {
+
+  // TODO:
+  // - move it to the boost_mp.h
+  // - can we use gcd only sometimes to save time?
+  const boost::multiprecision::cpp_int r = boost::multiprecision::gcd(a, b);
+  // std::cout << "r: " << r << std::endl;
+  // std::cout << "a: " << a << std::endl;
+  // std::cout << "b: " << b << std::endl;
+  a = a / r;
```

Comment:
It is a good habit to write `a /= r;` instead, so the compiler and library don't need to go out of their way to notice that it is the same `a`. Same with `>>=` elsewhere.

Reply:
True. I have fixed that.